tests/UnassignedAnalyserTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Tests\Qossmic\Deptrac;
+
+use PHPUnit\Framework\TestCase;
+use Qossmic\Deptrac\Configuration\Configuration;
+use Qossmic\Deptrac\ContainerBuilder;
+use Qossmic\Deptrac\UnassignedAnalyser;
+use Tests\Qossmic\Deptrac\Fixtures\LayerAnalyser\ClassBar;
+
+/**
+ * @covers \Qossmic\Deptrac\UnassignedAnalyser
+ */
+class UnassignedAnalyserTest extends TestCase
```

Comment:
Can you add another test case where a php internal class is used in the Fixture to make sure it is not returned by the analyzer to make sure the behavior does not change later if we change the implementation at some point.